content/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers.md

```diff
@@ -52,27 +52,24 @@ This approach can be especially useful for sending to Datadog logs that have hei
         module(load="imfile" PollingInterval="10") #needs to be done just once
         ```
 
-2. Create a `/etc/rsyslog.d/datadog.conf` file
+2. Create a `/etc/rsyslog.d/datadog.conf` file.  
+3. (Optional) Set the files to monitor. Add the following in `/etc/rsyslog.d/datadog.conf`.  
+    * **Rsyslog Version <8**.  
 
-3. (Optional) Set the file to monitor
-    Add the following in `/etc/rsyslog.d/datadog.conf`
-        * **Rsyslog Version <8**
-
-        ```
-        # Input for FILE1
-        $InputFileName /<path_to_file1>
-        $InputFileTag <app_name_of_file1>
-        $InputFileStateFile <unique_file_id1>
-        $InputFileSeverity info
-        $InputRunFileMonitor
-        ```
-
-        * **Rsyslog Version >= 8**
+    ```
+    # Input for FILE1
+    $InputFileName /<path_to_file1>
+    $InputFileTag <app_name_of_file1>
+    $InputFileStateFile <unique_file_id1>
+    $InputFileSeverity info
+    $InputRunFileMonitor
+    ```
+    * **Rsyslog Version >= 8**
 
-        ```
-        #For each file to send
-        input(type="imfile" ruleset="infiles" Tag="<app_name_of_file1>" File="<path_to_file1>" StateFile="<unique_file_id1>")
-        ```
+    ```
+    #For each file to send
```

Comment:
Could you please add a space after the comment like so: `# For each file to send`?

Reply:
👍